apps/ledger-live-mobile/src/families/ethereum/EthereumCustomFees/Ethereum1559CustomFees.tsx

```diff
@@ -131,7 +131,9 @@ const Ethereum1559CustomFees = ({
   useEffect(() => {
     if (!transaction) return;
     setTransaction(bridge.updateTransaction(transaction, transactionPatch));
-  }, [maxPriorityFeePerGas, maxFeePerGas, transactionPatch]);
+    // TODO: Some of the dependencies were ommitted, is that by choice ?
```

Comment:
It likely was, I would avoid changing the behaviour and put a TS ignore instead

Reply:
Noted, will fix